rules/go_rules.build_defs

```diff
@@ -1267,7 +1267,7 @@ def go_module(name:str='', module:str, version:str='', download:str='', deps:lis
             "go": f":{name}", # provide the filegroup otherwise exported deps don't work
             "go_src": download,
         },
-        labels = [f"go_package:{i}" for i in install],
+        labels = [f"go_package:{i}" for i in install] + [f"go_module:{module}"],
```

Comment:
I also needed a little more information on the `go_module()` rules to make my go deps thing work